x/mint/abci.go

```diff
@@ -85,6 +88,71 @@ func updateMinter(minter *types.Minter, blockTime sdk.Uint, newNormTime sdk.Dec,
 	return newlyMinted
 }
 
+// Returns the amount of tokens that should be minted by the integral formula
+// for the period between NormTimePassed and the timeInFuture.
+func predictMintedByIntegral(totalMinted sdk.Uint, normTimePassed, timeAhead sdk.Dec) (sdk.Uint, error) {
+	normTimeInFuture := normTimePassed.Add(timeAhead.Mul(types.NormMonthsRange))
```

Comment:
[here and below] use calcFunctionIncrement with nanoSecondsIn12Months = twelveMonths * nanoSecondsInMonth